src/scripts/main.js

```diff
@@ -1,3 +1,163 @@
 'use strict';
+// работа с ячейками
 
-// write code here
+const body = document.querySelector('body');
+const tbody = document.querySelector('tbody');
+
+tbody.addEventListener('click', (ev) => {
+  const activeRow = ev.target.closest('tr');
+
+  [...tbody.rows].forEach((row) => row.classList.remove('active'));
+  activeRow.classList.add('active');
+});
+
+body.insertAdjacentHTML('beforeend', `
+  <form class="new-employee-form" action="#" method="get">
+    <label>
+      Name:
+      <input
+        type="text"
+        name="name"
+        data-qa="name"
+        required
+      >
+    </label>
+    <label>
+      Position:
+      <input
+        type="text"
+        name="position"
+        data-qa="position"
+        required
+      >
+    </label>
+    <label>
+      Office:
+      <select
+        name="office"
+        name="office"
+        data-qa="office"
+        required
+      >
+        <option value="Tokyo">Tokyo</option>
+        <option value="Singapore">Singapore</option>
+        <option value="London">London</option>
+        <option value="New York">New York</option>
+        <option value="Edinburgh">Edinburgh</option>
+        <option value="San Francisco">San Francisco</option>
+      </select>
+    </label>
+    <label>
+      Age:
+      <input
+        type="number"
+        name="age"
+        data-qa="age"
+        required
+      >
+    </label>
+    <label>
+      Salary:
+      <input
+        type="number"
+        name="salary"
+        data-qa="salary"
+        required
+      >
+    </label>
+    <button class="button"> Save to table </button>
+  </form>
+`);
+
+function notification(type, title, text) {
+  body.insertAdjacentHTML('beforeend', `
+  <div
+    class="notification ${type}"
+    data-qa="notification"
+  >
+    <h1 class="title">
+      ${title}
+    </h1>
+    <p>
+      ${text}
+    </p>
+  </div>
+  `);
+
+  setTimeout(() =>
+    body.removeChild(document.querySelector('.notification')), 2000);
+};
+
+function strEdited(str) {
```

Comment:
Don't use name cuts
Also, if it is a function, start the name with a verb that explains what the function does. For example `editString`
But in your case, you can even specify: `capitalizeString`